site/_docs/reference.md

```diff
@@ -2850,6 +2850,7 @@ In the following:
 | b | REGEXP_SUBSTR(string, regexp [, position [, occurrence]]) | Synonym for REGEXP_EXTRACT
 | b m p s | REPEAT(string, integer)                  | Returns a string consisting of *string* repeated of *integer* times; returns an empty string if *integer* is less than 1
 | b m | REVERSE(string)                              | Returns *string* with the order of the characters reversed
+| s | REVERSE( string | array )                      | Returns *string* or *array* with the order of the characters reversed
```

Comment:
Returns *string* with the characters in reverse order or *array* with elements in reverse order

Reply:
Done, updated the description